```
/home/runner/work/mod-detailed-logging/mod-detailed-logging/modules/mod-detailed-logging/src/kill_stat_tracker.cpp:33:92: fatal error: no member named 'getLevel' in 'Player'; did you mean 'GetLevel'?
   33 |         killStream << player->GetName() << ", " << player->GetFaction() << ", " << player->getLevel() << ", " << player->GetMaxHealth() << ", " << player->GetHealth() << ", ";
      |                                                                                            ^~~~~~~~
      |                                                                                            GetLevel
/home/runner/work/mod-detailed-logging/mod-detailed-logging/src/server/game/Entities/Unit/Unit.h:1033:25: note: 'GetLevel' declared here
 1033 |     [[nodiscard]] uint8 GetLevel() const { return uint8(GetUInt32Value(UNIT_FIELD_LEVEL)); }
      |                         ^
1 error generated.
```

--- a/src/kill_stat_tracker.cpp
+++ b/src/kill_stat_tracker.cpp
@@ -30,7 +30,7 @@ void KillStatTracker::OnPlayerCreatureKilledByPet(Player* player, Creature* crea
         auto tm = *std::localtime(&t);
 
         killStream << std::put_time(&tm, "%d-%m-%Y %H-%M-%S") << ", ";
-        killStream << player->GetName() << ", " << player->GetFaction() << ", " << player->getLevel() << ", " << player->GetMaxHealth() << ", " << player->GetHealth() << ", ";
+        killStream << player->GetName() << ", " << player->GetFaction() << ", " << player->GetLevel() << ", " << player->GetMaxHealth() << ", " << player->GetHealth() << ", ";
         killStream << creature->GetName() << ", " << creature->GetFaction() << ", " << creature->GetMaxHealth() << ", ";
         killStream << player->GetZoneId() << ", " << player->GetAreaId() << ", " << player->IsGameMaster();
         killStream << "\n";

